modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/UpgradeSourceProcessorTest.java

```diff
@@ -164,6 +164,11 @@ public void testUpgradeJavaDLFolderMethodCheck() throws Exception {
 				"see LPS-194001."));
 	}
 
+	@Test
+	public void testUpgradeJavaExtractSetPortletCheck() throws Exception {
+		test("upgrade/UpgradeJavaExtractSetPortletCheck.testjava");
```

Comment:
Hi @MicaelleSilvaa, the same doesn't make sense to me. Could you rename it to `UpgradeJavaBasePanelAppExtendedClassesCheck`

Reply:
resolved